Hardware/Hardware/CardReader/CardBrand.swift

```diff
@@ -21,3 +21,64 @@ public enum CardBrand {
     /// An unknown card brand
     case unknown
 }
+
+extension CardBrand: CustomStringConvertible {
```

Comment:
I could see these strings in a more generic place for use by non card reader code, but this is a good start